<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaLibrary</string>
  </property>
  <property name="windowIcon">
   <iconset theme="mail-read" resource="Lib.qrc">
    <normaloff>:/icon/icon/books.png</normaloff>:/icon/icon/books.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox_datanode">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>11</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="currentText">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="placeholderText">
         <string>Type...</string>
        </property>
        <item>
         <property name="text">
          <string>Book</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Picture</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CD</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Author</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="placeholderText">
         <string>Search by name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>9</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>MediaLibrary</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionadd"/>
   <addaction name="actionCD"/>
   <addaction name="actionPic"/>
   <addaction name="actionUpdate"/>
   <addaction name="actionDel"/>
  </widget>
  <action name="actionadd">
   <property name="icon">
    <iconset resource="icon/icon2.qrc">
     <normaloff>:/icons/add-file.png</normaloff>:/icons/add-file.png</iconset>
   </property>
   <property name="text">
    <string>Book</string>
   </property>
   <property name="toolTip">
    <string>add new data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCD">
   <property name="icon">
    <iconset theme="accessories-calculator" resource="icon/icon2.qrc">
     <normaloff>:/icons/add-file.png</normaloff>:/icons/add-file.png</iconset>
   </property>
   <property name="text">
    <string>CD</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
    </font>
   </property>
  </action>
  <action name="actionPic">
   <property name="icon">
    <iconset resource="icon/icon2.qrc">
     <normaloff>:/icons/add-file.png</normaloff>:/icons/add-file.png</iconset>
   </property>
   <property name="text">
    <string>Pic</string>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/update.png</normaloff>:/icon/icon/update.png</iconset>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionDel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/delete.png</normaloff>:/icon/icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon/icon2.qrc"/>
  <include location="icon.qrc"/>
  <include location="Lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tableWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>actionDel</receiver>
   <slot>resetEnabled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
